CONTRIBUTING.md

```diff
@@ -22,6 +23,56 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 kubectl scale deployments/inflate --replicas=3
 ```
 
+### Use local environment instead of GitHub Codespaces. 
+> Tested environment : WSL2
+
+1. **Install tools**
+   * make
+     ```bash
+     sudo apt update && sudo apt install make -y
+     ```
+   * docker-cli
+     ```bash
+     curl -fsSL https://get.docker.com | bash -
+     ```
+   * kubectl
+     ```bash
+     curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && chmod +x kubectl && sudo mv kubectl /usr/local/bin
+     ```
+   * [helm](https://github.com/helm/helm/releases)
+     ```bash
+     curl -L https://get.helm.sh/helm-v3.14.3-linux-amd64.tar.gz | sudo tar xvfz - linux-amd64/helm && sudo mv linux-amd64/helm /usr/local/bin/helm && rm -rf linux-amd64
+     ```
+   * [golang](https://go.dev/dl/) > 1.21
+     ```bash
+     curl -L https://go.dev/dl/go1.22.1.linux-amd64.tar.gz | sudo tar xvzf - -C /usr/local/ && PATH=$PATH:/usr/local/go/bin
+     ```
+   * [yq](https://github.com/mikefarah/yq/releases) / [jq](https://github.com/jqlang/jq/releases)
+     ```bash
+     sudo apt update && sudo apt install jq -y
+     curl -Lo yq https://github.com/mikefarah/yq/releases/download/v4.43.1/yq_linux_amd64 && chmod +x yq && sudo mv yq /usr/local/bin
+     ```
+   * [skaffold](https://skaffold.dev/docs/install/#standalone-binary)
+     ```bash
+     curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v2.10.1/skaffold-linux-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin
+     ```
+   * [azure-cli](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+     ```bash
+     curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
```

Comment:
instead of this, I would prefer a makefile command to go alongside make toolchain that installs all the required binaries

Reply:
I added the install-tools parameter at the Makefile-az.mk.
Additionally, the introduction to tools that need to be installed has been removed.